kg_microbe/utils/s3_utils.py

```diff
@@ -107,72 +119,49 @@ def run_uniprot_api(show_status: bool) -> None:
     # Ensure the directory for storing Uniprot files exists
     Path(UNIPROT_S3_DIR).mkdir(parents=True, exist_ok=True)
 
-    organism_list = get_organism_list()
+    organism_ids = fetch_uniprot_reference_proteome_data()
 
-    # Process uniprot files
-    total_organisms = len(organism_list)
-    progress_class = tqdm if show_status else DummyTqdm
+    return organism_ids
 
-    # Iterate over organism IDs and fetch data from Uniprot
-    with progress_class(total=total_organisms, desc="Processing uniprot files") as progress:
```

Comment:
This allows us to turn off or turn on the progress bar if needed.  We should don't delete this entirely.

Reply:
Ok I wasn't sure how to get progress for this one, I suppose by page? Because it isn't by organism, it is only 1 query